### PR TITLE
Applications

### PR DESCRIPTION
**Pull request by sarynm12**
- fixed shelters model spec 
- all tests passing with 100% coverage

### CODE REVIEW

File: app/models/shelter.rb [hunk 1]
```diff
@@ -35,7 +31,11 @@ def average_review(shelter_id)
...
-    hi = Application.where("shelter_id = #{shelter_id}")
-    require "pry"; binding.pry
+    pet_id = Pet.where("shelter_id = #{shelter_id}").pluck(:id)
+    applications_array = []
+    pet_count = pet_id.each do |id|
+      applications_array << ApplicationsPets.where("pets_id = #{id}")
+    end
+    applications_array.flatten.count
   end
 end
```
Comment (lkriffell) [inline app/models/shelter.rb:41]: Nice. Looks like our number of apps method is functional now

File: app/models/shelter.rb [hunk 2]
```diff
@@ -14,14 +14,10 @@ def pets_pending
...
-      review.destroy
-    end
+    reviews.delete_all
     pets.each do |pet|
       app_pets = ApplicationsPets.where("pets_id = #{pet.id}")
-      app_pets.each do |app_pet|
-        app_pet.destroy
-      end
+      app_pets.delete_all
       pet.destroy
```
Comment (fieldstyler) [inline app/models/shelter.rb:21]: Solid refactor.

**Comment by lkriffell** [comment]
Reviewed by: Logan Riffell

File: spec/models/shelters_spec.rb [hunk 3]
```diff
@@ -84,10 +83,11 @@
                                   description: "I'm the best.")
       @app_pets_1 = ApplicationsPets.create!(applications_id: @app1.id, pets_id: @pet1.id)
       @app_pets_2 = ApplicationsPets.create!(applications_id: @app1.id, pets_id: @pet2.id)
+      #require "pry"; binding.pry
```
Comment (fieldstyler) [inline spec/models/shelters_spec.rb:86]: Take out pry

Comment (lkriffell) [inline spec/models/shelters_spec.rb:86]: Don't forget the pry

**Pull request merged**